fix broken links for docs - 4.x

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

cd into docs and run `yarn build` You will get errors of broken links in documentation. Discovered from #5296
```

[ERROR] Unable to build website for locale en.
[ERROR] Error: Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

- On source page path = /docs/api/modules/web3_eth_accounts:
   -> linking to web3_eth_accounts.md#signatureobject (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to web3_eth_accounts.md#keystore (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to ../classes/web3_eth_accounts.Wallet.md (resolved as: /docs/api/classes/web3_eth_accounts.Wallet.md)
   -> linking to ../interfaces/web3_eth_accounts.Web3Account.md (resolved as: /docs/api/interfaces/web3_eth_accounts.Web3Account.md)
   -> linking to web3_eth_accounts.md#cipher (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to web3_eth_accounts.md#scryptparams (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to web3_eth_accounts.md#pbkdf2sha256params (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to web3_utils.md#hexstring (resolved as: /docs/api/modules/web3_utils.md)
   -> linking to web3_eth_accounts.md#signresult (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to web3_eth_accounts.md#signtransactionresult (resolved as: /docs/api/modules/web3_eth_accounts.md)
   -> linking to web3_eth_accounts.md#cipheroptions (resolved as: /docs/api/modules/web3_eth_accounts.md)

    at reportMessage (/Users/alexluu/4.x/web3.js/docs/node_modules/@docusaurus/utils/lib/jsUtils.js:88:19)
    at handleBrokenLinks (/Users/alexluu/4.x/web3.js/docs/node_modules/@docusaurus/core/lib/server/brokenLinks.js:153:35)
    at async buildLocale (/Users/alexluu/4.x/web3.js/docs/node_modules/@docusaurus/core/lib/commands/build.js:176:5)
    at async tryToBuildLocale (/Users/alexluu/4.x/web3.js/docs/node_modules/@docusaurus/core/lib/commands/build.js:37:20)
    at async mapAsyncSequential (/Users/alexluu/4.x/web3.js/docs/node_modules/@docusaurus/utils/lib/jsUtils.js:36:24)
    at async build (/Users/alexluu/4.x/web3.js/docs/node_modules/@docusaurus/core/lib/commands/build.js:72:21)
```

### Expected Behavior

No broken links

### Steps to Reproduce

cd docs
yarn start

### Web3.js Version

4.x

### Environment

- Operating System: 
- Browser: 
- Node.js Version: 
- NPM Version: 


### Anything Else?

_No response_

--- a/docs/docusaurus.config.js
+++ b/docs/docusaurus.config.js
@@ -46,7 +46,7 @@ const config = {
 	url: 'https://docs.web3js.org',
 	baseUrl: '/',
 	onBrokenLinks: 'throw',
-	onBrokenMarkdownLinks: 'warn',
+	onBrokenMarkdownLinks: 'throw',
 	favicon: 'img/favicon.ico',
 
 	// GitHub pages deployment config.

--- a/docs/docusaurus.config.js
+++ b/docs/docusaurus.config.js
@@ -45,7 +45,7 @@ const config = {
 	tagline: 'The ultimate JavaScript library for Ethereum',
 	url: 'https://docs.web3js.org',
 	baseUrl: '/',
-	onBrokenLinks: 'warn',
+	onBrokenLinks: 'throw',
 	onBrokenMarkdownLinks: 'warn',
 	favicon: 'img/favicon.ico',
 

